Save Image as GIF doesn't work

```
An error occurred in a Save Image node:

OpenCV(4.6.0) D:\a\opencv-python\opencv-python\opencv\modules\imgcodecs\src\loadsave.cpp:730: error: (-2:Unspecified error) could not find a writer for the specified extension in function 'cv::imwrite_'

Input values:
• Image: RGB image 1024x1024
• Base Directory: "C:\\Users\\micha\\Desktop"
• Subdirectory Path: "asdasad"
• Image Name: "img"
• Image Extension: GIF
• Quality: 100
```

All other formats work.

Chainner was started with `npm start`, so the OpenCV/PIL version used is the one specified in `depencies.ts`.

--- a/backend/src/nodes/nodes/image/save_image.py
+++ b/backend/src/nodes/nodes/image/save_image.py
@@ -75,7 +75,7 @@ class ImWriteNode(NodeBase):
 
         os.makedirs(base_directory, exist_ok=True)
         # Any image not supported by cv2, will be handled by pillow.
-        if extension not in ["png", "jpg", "gif", "tiff", "webp"]:
+        if extension not in ["png", "jpg", "tiff", "webp"]:
             channels = get_h_w_c(img)[2]
             if channels == 1:
                 # PIL supports grayscale images just fine, so we don't need to do any conversion

